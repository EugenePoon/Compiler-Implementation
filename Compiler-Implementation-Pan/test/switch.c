int main(){
    int i;
    int n;
    i=0;
    n=1;
    switch(n){
        case 1:{i=n+n;i=i+5;}
        case 2:{i=i*2;}
        case 3:{i=i+1;break;}
        case 5:i=i+n*n;
    }
    print i;
}