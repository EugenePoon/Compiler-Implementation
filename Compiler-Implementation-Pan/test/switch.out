24 19 0 5 25 15 1 15 1 13 0 0 12 15 -1 13 0 1 1 0 1 12 15 -1 13 0 1 1 11 0 1 6 17 60 13 13 0 1 1 11 13 0 1 1 11 1 12 15 -1 13 13 11 0 5 1 12 15 -1 16 70 13 0 1 1 11 0 2 6 17 81 13 13 11 0 2 3 12 15 -1 16 91 13 0 1 1 11 0 3 6 17 102 13 13 11 0 1 1 12 15 -1 16 130 13 0 1 1 11 0 5 6 17 130 13 13 11 13 0 1 1 11 13 0 1 1 11 3 1 12 15 -1 13 11 22 21 2